server/models/Collective.js

```diff
@@ -1613,6 +1613,7 @@ export default function (Sequelize, DataTypes) {
     const updatedValues = {
       HostCollectiveId: hostCollective.id,
       hostFeePercent: hostCollective.hostFeePercent,
+      platformFeePercent: hostCollective.platformFeePercent !== null ? hostCollective.platformFeePercent : undefined,
```

Comment:
What is the expected behavior here? If `hostCollective.platformFeePercent` is `null` ?

We should make sure that it's reset to `OC_FEE_PERCENT`.

Reply:
I want to fall-back to the `beforeCreate` hook in the collective model.
That hook will make sure to set the right OC_FEE_PERCENT.

Reply:
At this stage, it's not a creation, it's an update.

Reply:
I'm using `undefined` because I don't want to convey the idea that this will be necessarily set to `null`.

Reply:
Nice, so undefined will make sure we're not updating its value, right?